tests/Unit/Models/UserStoryTest.php

```
@@ -343,9 +343,29 @@ function (string $sql, array $params) use ($stmt, &$capturedParams): \PDOStateme
                 return $stmt;
             }
         );
-        UserStory::markQualityFailed($db, 1, 3, 'exc:RuntimeException');
+        UserStory::markQualityFailed($db, 1, 5, 'exc:RuntimeException');
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\StratFlow\Models\UserStory' in method 'markQualityFailedRecordsExhaustedAttemptsAndError'. (undefined)

(StaticAccess)